Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileDeleted.yaml

```diff
@@ -63,6 +63,7 @@ ParserQuery: |
                     , Process = ActingProcessName
                     , Dvc = DvcHostname
                     , FilePath = TargetFilePath
+                | project-away EventData
```

Comment:
project-away also ParameterXml, RenderedDescription

Reply:
they do not appear in this schema. nothing else to project-away

Reply:
Not sure I understand. The Event table has those fields.

Reply:
you're right, i dont know why iv'e missed it. fixed